### PR TITLE
Update changelog 1.13.0-rc1

### PR DESCRIPTION
**Pull request by anwittin**
Please review the changelog to check that the right info was included.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,260 @@
...
+* **Azure Auth Rotate Root**: Add support for rotate root in Azure Auth engine [[GH-19077](https://github.com/hashicorp/vault/pull/19077)]
+* **Kubernetes Secrets Engine UI**: Kubernetes is now available in the UI as a supported secrets engine. [[GH-17893](https://github.com/hashicorp/vault/pull/17893)]
+* **New PKI UI**: Add beta support for new and improved PKI UI [[GH-18842](https://github.com/hashicorp/vault/pull/18842)]
+* **Server UDS Listener**: Adding listener to Vault server to serve http request via unix domain socket [[GH-18227](https://github.com/hashicorp/vault/pull/18227)]
+* **Transit managed keys**: The transit secrets engine now supports configuring and using managed keys
+* **VMSS Flex Authentication**: Adds support for Virtual Machine Scale Set Flex Authentication [[GH-19077](https://github.com/hashicorp/vault/pull/19077)]
+
+IMPROVEMENTS:
+
+* **Redis ElastiCache DB Engine**: Renamed configuration parameters for disambiguation; old parameters still supported for compatibility. [[GH-18752](https://github.com/hashicorp/vault/pull/18752)]
```
Comment (mladlow) [inline CHANGELOG.md:38]: @maxcoulombe - can you please adjust the changelog file before the GA? If this is a new feature, it should be in the features section, and if this is an improvement, it should use improvements formatting.

I believe you can submit a PR updating the file on the OSS 1.13 release branch.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,260 @@
...
+* ui: Fix bug where logging in via OIDC fails if browser is in fullscreen mode [[GH-19071](https://github.com/hashicorp/vault/pull/19071)]
+* ui: Fixes issue with not being able to download raft snapshot via service worker [[GH-17769](https://github.com/hashicorp/vault/pull/17769)]
+* ui: Fixes oidc/jwt login issue with alternate mount path and jwt login via mount path tab [[GH-17661](https://github.com/hashicorp/vault/pull/17661)]
+* ui: Remove default value of 30 to TtlPicker2 if no value is passed in. [[GH-17376](https://github.com/hashicorp/vault/pull/17376)]
+* ui: allow selection of "default" for ssh algorithm_signer in web interface [[GH-17894](https://github.com/hashicorp/vault/pull/17894)]
+* ui: cleanup unsaved auth method ember data record when navigating away from mount backend form [[GH-18651](https://github.com/hashicorp/vault/pull/18651)]
+* ui: fix entity policies list link to policy show page [[GH-17950](https://github.com/hashicorp/vault/pull/17950)]
+* ui: fixes query parameters not passed in api explorer test requests [[GH-18743](https://github.com/hashicorp/vault/pull/18743)]
+
 ## 1.13.0
```
Comment (mladlow) [inline CHANGELOG.md:258]: @anwittin we should replace the 1.13.0 preview with the RC content.

Comment (anwittin) [reply CHANGELOG.md:258]: @mladlow I figured that was what we should do, but wasn't sure. I will remove the 1.13.0 unreleased section

Comment (anwittin) [reply CHANGELOG.md]: Section removed

**Review by mladlow — APPROVED** [review]
Looks good! I've made a few requests to engineers that might show up as changes for the GA.

I have mostly reviewed the proper formatting of the "new features" section. I don't typically review the changelog extremely closely.

**Pull request merged**